### PR TITLE
Allow IDE typehinting/clickability

### PR DESCRIPTION
**Pull request by dereuromark**

### CODE REVIEW

File: config/routes.php [hunk 1]
```diff
@@ -44,6 +44,8 @@
 Router::defaultRouteClass(DashedRoute::class);
 
 Router::scope('/', function (RouteBuilder $routes) {
+    /* @var \Cake\Routing\RouteBuilder $routes */
```
Comment (lorenzo) [inline config/routes.php]: WAT

Comment (dereuromark) [inline config/routes.php]: What class should it be?

**Comment by dereuromark** [comment]
Ah never mind. I copy and pasted from my app Code. Seems like the typehint is now already finally included :) im the Funktion Signature
Didn't see that.

File: config/routes.php (hunk 1 above)
Comment (lorenzo) [inline config/routes.php]: No, I just find it funny that the variable is type hinted but you sill need to add another hint in th next line

Comment (dereuromark) [inline config/routes.php]: I just had outdated app Code ;) and it was late.

**Pull request closed without merging**